core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -1684,6 +1684,18 @@ public boolean isDistributedPruningEnabled(String dbName, String tableName) {
     return isServerEnabledByUser;
   }
 
+  /**
+   * Check whether the pre-priming is enabled by the user or not.
+   */
+  public boolean isIndexServerPrePrimingEnabled() {
+    // Check if user has enabled/disabled the use of pre-priming for index server
```

Comment:
remove this comment, and update the method comment

Reply:
done